### ISSUE TITLE
CMI 9.7.8.0 Folia Region Scheduling Error

### ISSUE THREAD

**Issue by YusakiDev** [Zrips/CMI#9336]
### Description of Issue

folia region access error spam

### Version Information

```shell
[13:51:29 INFO]: --------------------------------------------------
[13:51:29 INFO]: CMI: 9.7.8.0 Velocity CMIB  SqLite
[13:51:29 INFO]: CMILib: 1.5.2.2 -> 1.5.2.3
[13:51:29 INFO]: Server: Folia 1.21.1-R0.1-SNAPSHOT-
[13:51:29 INFO]: CMI economy: Disabled Vault: 1.7.3-CMI CMI Chat: Disabled 
[13:51:29 INFO]: Modules -> 33 enabled 34 disabled: spawnerProximity, cuffed, tablist, versionCheck, armorEffects, moneyCheque, skin, noTarget, rideAnimation, bossBarCompass, coloredArmor, chatBubble, votifier, shulkerBackpack, headDrop, spawnerCharge, interactiveCommands, namePlates, homeInteractions, launchPad, elytraExploit, anvilRenameColor, jail, spawners, paintingEditor, playerChatTag, durabilityLoss, kits, disabledEnchants, worldLimitsElytra, ranks, firstJoinMessages, sitAnimation, totemBehavior
[13:51:29 INFO]: --------------------------------------------------
```

### Errors

```shell
https://mclo.gs/c9ou4bA
```

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by asygamesperson** [comment]
Also I get this error on folia 1.21.4 https://paste.helpch.at/todaxupara.swift

**Comment by Zrips** [comment]
@YusakiDev 
Your issue will be fixed with next update
@asygamesperson 
You used still unsupported CMI version which created your error message, this will be addressed with next release. Just don't forget to update CMILib as it will include few fixes for 1.21.4 servers
